aws/aws.go

```diff
@@ -272,6 +280,54 @@ func deleteVolumes(ec2Clients map[string]ec2Client, volumes []*types.Disk) []err
 	return errs
 }
 
+func deleteImages(ec2Clients map[string]ec2Client, images []*types.Image) []error {
+	regionImages := map[string][]*types.Image{}
+	for _, image := range images {
+		if image.CloudType == types.AWS {
+			regionImages[image.Region] = append(regionImages[image.Region], image)
+		}
+	}
+	log.Debugf("[AWS] Delete images: %v", regionImages)
+
+	wg := sync.WaitGroup{}
+	wg.Add(len(regionImages))
+	errChan := make(chan error)
+	sem := make(chan bool, 10)
+
+	for r, v := range regionImages {
+		go func(ec2Client ec2Client, region string, images []*types.Image) {
+			sem <- true
```

Comment:
What is this sem used for?

Reply:
To control number of parallel requests. Otherwise we send 700 requests to for example Azure, and i guess it doesn't like that

Reply:
I see, thanks.